app/scripts/modules/core/src/routing/uirouter.stateEvents.shim.js

```diff
@@ -1,146 +0,0 @@
-/**
```

Comment:
There are a lot of `$stateChangeSuccess` and `$stateChangeStart` event subscriptions throughout the application. What fires those events now?